src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -81,19 +81,20 @@
                                 <CheckBox x:Uid="FancyZones_ShowZonesOnAllMonitorsCheckBoxControl" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.ShowOnAllMonitors}" Margin="{StaticResource ExpanderSettingMargin}"  />
                                 <Rectangle Style="{StaticResource ExpanderSeparatorStyle}" />
                                 <CheckBox IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.SpanZonesAcrossMonitors}" 
-                                          Margin="{StaticResource ExpanderSettingMargin}"
+                                          Margin="56, 0, 40, 16"
```

Comment:
The same CheckBox margin is also on the PowerLauncher page. Maybe it's worth doing to create a resource for it. Isn't it?

Reply:
I think the best option would be to make a CheckBoxWithDescription control. We can then set the correct margins and also handle the AutomationProperties.Name.